app/controllers/vendors_controller.rb

```diff
@@ -0,0 +1,61 @@
+class VendorsController < ApplicationController
+
+  def index
+    if params[:market_id] == nil
+      @vendors = Vendor.all
+    else
+      @vendors = Vendor.where(market_id: params[:market_id])
+      @market_edit = true
+    end
+  end
+
+  def show
+    @vendor = Vendor.find(params[:id])
+    @vendor_sales = (@vendor.sales.sum(:amount))/100.to_f
+
+    month_number = Time.now.strftime("%m").to_i
+    month_beginning = DateTime.new(Date.today.year, month_number)
+    month_ending = month_beginning.end_of_month
+
+    @monthly_sales = Sale.where(:purchase_time => month_beginning..month_ending).where(:vendor_id => @vendor.id)
+    @monthly_sales_total = (@monthly_sales.sum(:amount))/100.to_f
+    # @monthly_sales_transactions = (@monthly_sales)
```

Comment:
Don't forget to remove commented out code that you aren't going to use when you do your PR.